pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -34,6 +34,34 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	// The constant is used if sync_nodes_status=False. NodeController will not proactively
+	// sync node status in this case, but will monitor node status updated from kubelet. If
+	// it doesn't receive update for this amount of time, it will start posting node NotReady
```

Comment:
NotReady doesn't actually exist in the master, only in kubectl resource printer (unless I am misunderstanding).  So Instead of saying "NotReady condition" you should say "NodeReady == ConditionNone" (or whatever the appropriate NodeConditionType is). Please fix this everywhere you currently use NotReady in this file.